Python/books.md

```diff
@@ -0,0 +1,4 @@
+Learn Python the Hard Way | http://learnpythonthehardway.org/

```

Comment:
Please follow list style for the formatting, check this for reference https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/Useful_links.md